helm-charts/aerospike-restore/templates/aerospike-restore-cr.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: asdb.aerospike.com/v1beta1
+kind: AerospikeRestore
+metadata:
+  name: {{ template "aerospike-restore.commonName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "aerospike-restore.labels" . | nindent 4 }}
+    {{- with .Values.customLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  # Aerospike Backup Service reference
+  backupService:
```

Comment:
can we use {{ .Values.backupService }} directly instead of breaking into fields, so that for new fields in the future, we don't have to change.